### ISSUE TITLE
Release 7 | ROT/SP release requests

### ISSUE THREAD

**Issue by morlandi7** [oxidecomputer/hubris#1586]
Please link any changes requiring an ROT/SP release to be included in the next customer release

```[tasklist]
### ROT/SP Updates
- [ ] Add a draft title or issue reference here
- [ ] https://github.com/oxidecomputer/hubris/issues/1625
- [ ] https://github.com/oxidecomputer/hubris/pull/1627
```

**Comment by rmustacc** [comment]
Please note #1625 is a hard blocker and no releases can be cut until it is addressed as it breaks the use of the T6 on Gimlet.

**Comment by hawkw** [comment]
I've added PR #1627, which fixes #1625.
